leaderboardsgg/leaderboard-site
# LBGG[Nuxt3]: Setup API

## What

This PR connects the frontend to the backend, via some new composables.

## Link to Issue

N/A

## Acceptance

### Steps for testing

- Setup the backend repo, and make sure the local address is set in your `.env`
- Start up the frontend
- [ ] Register a new user
- [ ] Log in with the user

### Images

| • | Mobile | Tablet | Desktop |
| --- | --- | --- | --- |
| `SiteNavbar` when `loggedIn` | ![image](https://user-images.githubusercontent.com/17736056/173991344-2258f91b-176a-40f2-9586-be14b0736377.png) | ![image](https://user-images.githubusercontent.com/17736056/173991299-0666e6cd-d3e8-4aec-8416-608f94c1f997.png) | ![image](https://user-images.githubusercontent.com/17736056/173991247-0c38a002-20a1-45e2-93f3-8a9c26b7096c.png) |
